actions/src/utils.js

```diff
@@ -0,0 +1,26 @@
+const { context } = require('@actions/github');
+
+/**
+ * This function pings and assigns all pending reviewers to a pull request.
+ * @param {import('@octokit/rest').Octokit} octokit
+ * @param {import('@octokit/rest').Octokit.PullsGetResponse} pullRequest
+ * @param {string[]} assignees
+ * @param {string} comment
+ */
+const pingAndAssignUsers = async (octokit, pullRequest, assignees, comment) => {
```

Comment:
This function doesn't really ping users, it just creates comment (which might contain entirely different content).

Reply:
Yes, but it also assigns the users. Comments can be created directly. This function helps when we want to make the comment and also assign users at the same time.